standard_reports/expiring-items/2_3/src/template.html

```diff
@@ -21,45 +26,38 @@
         <td>{{stockLine.item.code}}</td>
         <td>{{stockLine.item.name}}</td>
         <td class="expiry-date">
-          {% if stockLine.daysUntilExpired %}
-            {% if stockLine.daysUntilExpired <= 0 %}
-              <span class="expired">{{t(k="report.expired", f="Expired")}}</span>
-            {% elif stockLine.daysUntilExpired <= arguments.monthsItemsExpire * 30 %}
-              <span class="soon">{{ stockLine.daysUntilExpired }}</span>
-            {% else %}
-                <span class="all-good">{{ stockLine.daysUntilExpired }}</span>
-            {% endif %}
+          {% if stockLine.daysUntilExpired %} {% if stockLine.daysUntilExpired
+          <= 0 %}
+          <span class="expired">{{t(k="report.expired", f="Expired")}}</span>
```

Comment:
I haven't tested this report, I wonder if we want it with the new version? Hard to know, what to do!